kettle/src/main/resources/standalone/configuration/hawkular-realm.json

```diff
@@ -57,17 +57,19 @@
       "secret" : "${uuid.hawkular.accounts.backend}"
     },
     {
-      "name": "hawkular-accounts-ui",
+      "name": "hawkular-ui",
       "enabled": true,
       "bearerOnly" : false,
       "publicClient": true,
-      "secret": "${uuid.hawkular.accounts.ui}",
+      "secret": "${uuid.hawkular.ui}",
       "fullScopeAllowed" : false,
       "redirectUris": [
+        "http://localhost:8080/*",
```

Comment:
Also that does not help. Just deploy that thing via `docker run -P ....  ` and you will get a "random" port and also a "random" IP depending on where you deploy it.

In JavaScript one can apparently use window.location() or such. And http headers seem to carry the "Origin" header (from Cors?) that gives the values.